pom.xml

```diff
@@ -30,7 +30,7 @@
         <docker.file>Dockerfile.${docker.os_type}</docker.file>
         <docker.tag>${project.version}-${docker.os_type}</docker.tag>
         <!-- Versions-->
-        <ubi.image.version>8.5-240</ubi.image.version>
+        <ubi.image.version>8.5-240.1648458092</ubi.image.version>
```

Comment:
For my own undestanding: Where does this version number come from?

Reply:
From DockerHub for redhat/ubi8-minimal: https://hub.docker.com/r/redhat/ubi8-minimal/tags

Also here's the section in the release troubleshooting guide describing why we make this update, basically we always run a `yum check-update` since customers want us to be on the latest version, and pin the version for reproducibility reasons: https://confluentinc.atlassian.net/wiki/spaces/TOOLS/pages/2565472764/Troubleshooting#The-command-'/bin/sh--c-yum-check-update-||-%22${SKIP_SECURITY_UPDATE_CHECK}%22'-returned-a-non-zero-code:-1in-common-docker